### PR TITLE
Update Deep_Learning_Project-Pytorch.ipynb

### PR DESCRIPTION
**Pull request by bobbleoxs**
address two small error messages: 

1. invalid index of a 0-dim tensor;
2. implicit dimension choice for softmax

### CODE REVIEW

**Comment by bobbleoxs** [comment]
warning errors relating to pytorch deprecate issues. they are very small but took me a while to figure out where to change them. hopefully will help others who follow the tutorial. thanks

**Comment by bobbleoxs** [comment]
sorry I think I submitted the same thing twice.

**Comment by Spandan-Madan** [comment]
Hi, thanks for contributing! Were these deprecations leading to warnings only? or did the code break as well?

**Comment by bobbleoxs** [comment]
apologies for late response.

the loss.item() did break my code but the dim=1 is just a deprecation from
memory.

cheers:-)
